test/IISExpress.FunctionalTests/InProcess/AppOfflineTests.cs

```diff
@@ -19,55 +19,75 @@ namespace Microsoft.AspNetCore.Server.IIS.FunctionalTests.Inprocess
     public class AppOfflineTests : IISFunctionalTestBase
     {
         // TODO these will differ between IIS and IISExpress
-        [ConditionalFact]
-        public async Task AppOfflineDroppedWhileSiteIsDown_SiteReturns503()
+        [ConditionalTheory]
```

Comment:
File an issue to make IIS app_offline tests.

Reply:
https://github.com/aspnet/IISIntegration/issues/1026